WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -95,6 +95,8 @@ - (BOOL)hidesBottomBarWhenPushed
 
 - (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
 {
+    [super willAnimateRotationToInterfaceOrientation:toInterfaceOrientation duration:duration];
```

Comment:
Just a though, `willAnimateRotationToInterfaceOrientation:duration:` is deprecated in iOS8.  Would it make sense to replace it with `viewWillTransitionToSize:withTransitionCoordinator:` as a part of this change?